routes/index.js

```diff
@@ -21,6 +22,7 @@ router.delete('/question/:id', auth_token, deleteQuestion.deleteQuestion);
 router.post('/answer', auth_token, postAnswer.createAnswer);
 router.get('/answer/:question_id', auth_token, viewAnswer.viewAnswer);
 router.put('/answer/:answer_id', auth_token, acceptAnswer.acceptAnswer);
+router.post('/comment/:id', auth_token, commentAnswer.commentAnswer);
 // router.post('/users', createUserController);
```

Comment:
Since the user is submitting his/her comment on an answer then the route should be like either

1. 

```javascript
router.post('/answer/:answer_id/comment, auth_token, commentAnswer.commentAnswer);
```
So when calling the API, the user will need to pass only the `comment_text` in the request body.

Then in the `commentAnswer` controller, you can get the `answer_id` by doing 
```javascript
const answer_id = req.params.answer_id; or const { answer_id} = req.params;
```

2. 
if you want to pass the `answer_id` in the request body, then the route should be 
```javascript
router.post('/comment, auth_token, commentAnswer.commentAnswer);
```
So when calling the API, the user will need to pass both `comment_text` and `answer_id` in the request body.

Then in the `commentAnswer` controller, you can get the `answer_id` by doing 
```javascript
const {answer_id, comment_text} = req.body;
```